src/main/java/ch/njol/skript/Skript.java

```diff
@@ -923,6 +927,24 @@ public void run() {
 			}
 		}, this);
 
+		// Send a warning to console when the plugin is reloaded
+		Bukkit.getPluginManager().registerEvents(new Listener() {
+			@EventHandler
+			public void onServerReload(ServerLoadEvent event) {
+				if ((event.getType() != ServerLoadEvent.LoadType.RELOAD)) return;
+
+				for (Player player : Bukkit.getOnlinePlayers()) {
```

Comment:
There is `Bukkit#getOperators()` method which returns offlineplayer, can then use `OfflinePlayer#getPlayer` and `OfflinePlayer#isOnline` to avoid iterating over every single player on the server.